packages/components/src/components/carousel/carousel-utils.spec.ts

```diff
@@ -468,3 +469,15 @@ describe('updatePagination()', () => {
     expect(el.children[3].outerHTML).toBe(bulletActiveMarkup);
   });
 });
+
+describe('getLangDirection()', () => {
```

Comment:
Does only test a part of the function currently

Reply:
Updated. Should now test all three scenarios.